src/components/FileUpload/FileUpload.tsx

```
@@ -0,0 +1,628 @@
+  TableRow,
+  Divider,
+  CircularProgress,
+} from '@mui/material';
+import {
+  CloudUpload,
+  Delete,
+  Clear,
+  Description,
+  InsertDriveFile,
+  CheckCircle,
+  Error,
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)